docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2423,6 +2424,14 @@ definitions:
           specified when creating subscription and sending it may result in a client error.
         format: RFC 3339 date-time
         example: '1996-12-19T16:39:57-08:00'
+      updated_at:
+        type: string
+        readOnly: true
+        description: |
+          Timestamp of last updation of the subscription. This is generated by Nakadi. It should not be
```

Comment:
I'm not sure if "updation" is a correct word. 
@lmontrieux?

Reply:
Definitely not a word :)
You're looking for `update`

Reply:
@Kunal-Jha please address this problem.